### ISSUE TITLE
Notifications Monitor Zero ETV and Keyword Items Text Difficult to Read

### ISSUE THREAD

**Issue by ooglek** [FMaz008/VineHelper#158]
The --link-dark CSS theme `.theme-default` combined with the Zero ETV Green background color makes the text of the item nearly impossible to read.

https://github.com/FMaz008/VineHelper/blob/15627d861182889c190539cf366329c4bf9549ec/resource/css/vinehelper.css#L74

<img width="511" alt="Screenshot 2024-10-22 at 12 07 36 PM" src="https://github.com/user-attachments/assets/3c338337-c1cd-43c9-bd24-eec9a9e43834">

Suggestion:

- For a Zero ETV Item, change the text `color` to white in `div.vh-notification-content` (currently set in vinehelper.css .theme-default)
- For a KW Item, change the text `color` to black or green for ALL text, including the ETV and Timestamp

<img width="327" alt="Screenshot 2024-10-22 at 12 09 09 PM" src="https://github.com/user-attachments/assets/99015e5c-1c4d-45bd-8f10-24b68ab9fe27">

**Comment by FMaz008** [comment]
Yeah I messed up when I assigned the default color for 0ETV. In the upcoming version links will default to the notmal blue. Now sure why some browser show them green and some blue.

https://github.com/FMaz008/VineHelper/commit/15627d861182889c190539cf366329c4bf9549ec

In the mean time you can change the background colors undet the Notification tabs in the settings

**Comment by ooglek** [comment]
This is probably why it is green:
<img width="352" alt="image" src="https://github.com/user-attachments/assets/a1186629-8ffa-4a65-ae0e-3964d5230947">

**Comment by FMaz008** [comment]
> This is probably why it is green: <img alt="image" width="352" src="https://private-user-images.githubusercontent.com/100755/378944423-a1186629-8ffa-4a65-ae0e-3964d5230947.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk2MTY5MzMsIm5iZiI6MTcyOTYxNjYzMywicGF0aCI6Ii8xMDA3NTUvMzc4OTQ0NDIzLWExMTg2NjI5LThmZmEtNGE2NS1hZTBlLTM5NjRkNTIzMDk0Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyMlQxNzAzNTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMDZhNzdiNzY3YTI3MWM5NDY2N2MzZjk3Y2MzMDA2YWZjNWYxYzA1MzBkYzRhOWM0Yjc5YzQ0NDdjYzNiNTVmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.zXehg_tgrtgwBHTiLJsYMmis_2D2_OydQCDQgVtdQ0A">

Ah yeah that would explain why I would not see the text as green on my end... dark mode. Good find. I'll remove that.

**Comment by FMaz008** [comment]
Should be fixed in the upcoming version
